src/gui/about_imp.h

```diff
@@ -36,6 +36,11 @@
 #include "base/utils/misc.h"
 #include "base/unicodestrings.h"
 
+#define SIXTYFOUR ""
```

Comment:
@Chocobo1 

Usually applications omit showing 32bitness - don't know why. Perhaps there's some logic to it but I'd do the same.